### ISSUE TITLE
[Back] Mettre à jour les stats publiques

### ISSUE THREAD

**Issue by hmeneuvrier** [MTES-MCT/histologe#1550]

**Comment by hmeneuvrier** [comment]
@mathildepoulpux @arnaudchouk on aura besoin de vos lumières pour savoir comment on regroupe les familles de désordres entre anciens et nouveaux signalements

**Comment by arnaudchouk** [comment]
c est 2 visions totalement différentes, ca va être complexe, je ne suis même pas sur que ce soit faisable sans très approximatif
je ne sais pas ce qu en dit @mathildepoulpux

**Comment by mathildepoulpux** [comment]
Oui c'est vraiment pas la même chose... 

Je vous propose : 
- Remplacer ce graph par un camembert "Bâtiment / logement" (là je peux faire la liste pour les désordres de l'ancien form) 
- Ajouter un graph "Les désordres les plus courants dans le logement" avec les 5 désordres les + renseignés pour cette partie
- Ajouter un graph "Les désordres les plus courants du bâtiment" avec les 5 désordres les + renseignés pour cette partie

Pour les 2 nouveaux graph : soit on accepte qu'il y aura un mélange d'intitulés entre ancien et nouveau form pour la partie "Total".
Soit on propose que 2024 et on ajoute une explication pour dire que ces stats sont nouvelles et dispo seulement depuis la mise à jour du service.

@arnaudchouk t'en penses quoi ?

**Comment by arnaudchouk** [comment]
Et est ce qu on peut envisager les 2 affichages (ancien/nouveau)?
avec un switch de l un à l autre sur la base de la date de migration (avant/apres )? :/

**Comment by mathildepoulpux** [comment]
On peut, mais ça fait un nouveau comportement. Là on raison en année en cours / total.

Juste pour ma compréhension, ça va être intéressant pour quels visiteurs de cette page d'avoir les 2 affichés ? 
Et est-ce que les données concernant l'ancien formulaire sont amenées à disparaître (genre est-ce que ce sera encore pertinent de les afficher dans 6 mois, 1 an...) ?

En tous cas, si on veut garder toutes les info :
On peut prendre la 2eme solution expliquée avant (avec les data du nouveau formulaire et on met un switch 2024 / total) + afficher un graph sans switch pour tout ce qu'il y avait avant la migration avec une phrase pour expliquer.

**Comment by arnaudchouk** [comment]
t as raison je basais ma reflexion sur les stats consultees par les connectés mais c est encore different des stats publiques

**Comment by mathildepoulpux** [comment]
## Mise à jour des stats publiques
Suite à la mise en ligne du nouveau formulaire, les statistiques publiques doivent être revues. 
Pour améliorer le produit et l'accessibilité, on en profite pour revoir les graphiques de la page en utilisant les composants du DSFR.

- [ ] Revoir les graph et la carte en utilisant les composants [DSFR charts](https://gouvernementfr.github.io/dsfr-chart/)
- [ ] Passer le donut "Motif de clôture" en barres verticales
- [ ] Supprimer le graph barres verticales "RÉPARTITION PAR FAMILLE DE DÉSORDRES"
- [ ] Ajouter un diagramme circulaire plein "DÉSORDRES PAR CATÉGORIE" qui reprend la proportion désordres par zone (logement ou bâtiment) -> voir le tableau pour la catégorisation des désordres de l'ancien formulaire
- [ ] Ajouter un diagramme circulaire creux (donut) "LOGEMENT : DÉSORDRES LES PLUS COURANTS*" : qui affiche les 5 désordres les plus renseignés pour le logement depuis la mise en ligne uniquement
- [ ] Ajouter un diagramme circulaire creux (donut) "BÂTIMENT : DÉSORDRES LES PLUS COURANTS*" : qui affiche les 5 désordres les plus renseignés pour le bâtiment depuis la mise en ligne uniquement
- [ ] Pour ces 2 derniers diagrammes, l'élément "Total" est inactif -> on affiche que les résultats depuis la mise en ligne et on met une astérisque en dessous

### Ressources
- Dans [ce tableau](https://docs.google.com/spreadsheets/d/1Asns_vsBs1Lf8UFMz_RcH0s5WOSkKXy9DXfds4cOfr8/edit#gid=1335478415) la catégorisation des critères de l'ancien formulaire par zone (logement ou bâtiment).
- Le [prototype de la nouvelle page](https://xd.adobe.com/view/8959c3fa-ca2e-4042-9983-5db664d3009e-dc0a/screen/c61af72f-bd7c-4361-a079-db7b1cba595e/) dispo sur Adobe Xd.

**Comment by arnaudchouk** [comment]
sur la page 
je propose dans la partie synthese en haut
d ajouter 
Tx de signalements refusés qui est Nombre de signalements Refusés / Signalements Enregistrés (ie sans import ni archivé)
Ensuite garder le reste
Peut etre renommer Tx de prise en charge par Tx de dossiers acceptés pris en charge par les partenaires
@mathildepoulpux ?

**Comment by mathildepoulpux** [comment]
Je suis pas bien sûr qu'on comprenne "taux de dossiers acceptés pris en charge par les partenaires" (la différence entre acceptés et pris en charge est pas clair quand tu viens de l'extérieur)
Ca te va si on met `Taux de dossiers en cours d'instruction` (ou de traitement)

**Comment by arnaudchouk** [comment]
je suis ok c est pas ideal
la notion d instruction me convient
mais si en cours d instruction, on change de principe car on retire les clotures
mais ca peut donner une meilleure comprehension
ca ferait 
signalements enregistres
signalements refuses
signalements en cours d instruction
signalements clotures
et le delta qui reste, ce serait les signalements encore en statut nouveau

**Comment by mathildepoulpux** [comment]
Oui, et les nouveaux on les voit de toute façon dans le camembert de la répartition par statut

**Comment by arnaudchouk** [comment]
ok donc: 

signalements enregistres : total enregistres - Archivés - Importés
signalements refuses : total des refus  / signalements enregistes
signalements en cours d instruction : total des signalements en cours  / signalements enregistes
signalements clotures : total des clotures / signalements enregistes

**Comment by hmeneuvrier** [comment]
Je vais traiter ce ticket, mais sans la partie dsfr charts qui est externalisée dans le ticket https://github.com/MTES-MCT/histologe/issues/2339 et sur laquelle @emilschn a passé beaucoup de temps avec une conclusion plutôt défavorable

**Comment by hmeneuvrier** [comment]
@mathildepoulpux 
pour

> Ajouter un diagramme circulaire creux (donut) "LOGEMENT : DÉSORDRES LES PLUS COURANTS*" : qui affiche les 5 désordres les plus renseignés pour le logement depuis la mise en ligne uniquement
> Ajouter un diagramme circulaire creux (donut) "BÂTIMENT : DÉSORDRES LES PLUS COURANTS*" : qui affiche les 5 désordres les plus renseignés pour le bâtiment depuis la mise en ligne uniquement

nous sommes au niveau du desordreCritere, n'est-ce pas ? (par exemple `Il n'y a pas de portes coupe feu pour les caves, ou pour accéder aux caves.` plustôt que la catégorie `Risque incendie` ou la catégorie BO `Bâtiment risques particuliers` ou que la précision qui là en l'occurrence est la même chose que le critère)

**Comment by mathildepoulpux** [comment]
Oui c'est bien desordreCritere

**Comment by hmeneuvrier** [comment]
Reste à faire : 

- [ ] Vérifier `Dans la partie "Histologe en quelques chiffres", remplacer la carte taux de prise en charge par une carte Taux de signalements en cours d'instruction qui correspond au pourcentage de signalements en cours / nbr de signalements enregistrés (sans les archivés et importés)`
- [ ] Ajouter un diagramme circulaire plein "DÉSORDRES PAR CATÉGORIE" qui reprend la proportion désordres par zone (logement ou bâtiment) -> voir le tableau pour la catégorisation des désordres de l'ancien formulaire
- [ ] Retour à la ligne pour les labels des donuts de désordres

**Comment by hmeneuvrier** [comment]
` Dans la partie "Histologe en quelques chiffres", remplacer la carte taux de prise en charge par une carte Taux de signalements en cours d'instruction qui correspond au pourcentage de signalements en cours / nbr de signalements enregistrés (sans les archivés et importés)` déjà fait là https://github.com/MTES-MCT/histologe/issues/2288
